app/src/main/java/com/pavesid/androidacademy/ui/MoviesViewModel.kt

```diff
@@ -29,7 +29,7 @@ class MoviesViewModel @ViewModelInject constructor(
         Timber.d(throwable)
     }
 
-    init {
+    fun getMovies() {
```

Comment:
`get` means that this function should return something.  
This function actually does side-effect. 
So I recommend you to rename it to `loadMovies()`

Reply:
Done.